src/base/snark0.ml

```diff
@@ -52,6 +52,9 @@ struct
     let field_vars = Array.map fields ~f:(fun x -> Cvar0.Constant x) in
     typ.var_of_fields (field_vars, aux)
 
+  (* to pass to Utils.Make *)
+  module As_prover_old = As_prover
+
```

Comment:
Not sure this alias is useful (see next comment)

Reply:
Will remove.